### PR TITLE
Upgraded latest axios version bump

### PR DESCRIPTION
**Pull request by sunil-lakshman**

### CODE REVIEW

**Review by aman19K — APPROVED** [review]

File: packages/contentstack-migration/package.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
   "name": "@contentstack/cli-migration",
-  "version": "1.6.1",
+  "version": "1.6.2",
   "author": "@contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "dependencies": {
     "@contentstack/cli-command": "~1.3.0",
-    "@contentstack/cli-utilities": "~1.7.1",
+    "@contentstack/cli-utilities": "~1.7.2",
```
Comment (aman19K) [inline packages/contentstack-migration/package.json:8]: @sunil-lakshman , for internal dependency we don't need to do version bump. pls check with @cs-raj

File: packages/contentstack-migrate-rte/package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@contentstack/cli-cm-migrate-rte",
   "description": "Contentstack CLI plugin to migrate HTML RTE to JSON RTE",
-  "version": "1.4.18",
+  "version": "1.4.19",
```
Comment (aman19K) [inline packages/contentstack-migrate-rte/package.json:4]: @sunil-lakshman , same apply here https://github.com/contentstack/cli/pull/1525/files#r1726703685

File: packages/contentstack-export-to-csv/package.json [hunk 3]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@contentstack/cli-cm-export-to-csv",
   "description": "Export entities to csv",
-  "version": "1.7.2",
+  "version": "1.7.3",
```
Comment (aman19K) [inline packages/contentstack-export-to-csv/package.json:4]: @sunil-lakshman , https://github.com/contentstack/cli/pull/1525/files#r1726703685

File: packages/contentstack-clone/package.json [hunk 4]
```diff
@@ -1,15 +1,15 @@
 {
   "name": "@contentstack/cli-cm-clone",
   "description": "Contentstack stack clone plugin",
-  "version": "1.10.7",
+  "version": "1.10.8",
```
Comment (aman19K) [inline packages/contentstack-clone/package.json:4]: @sunil-lakshman , https://github.com/contentstack/cli/pull/1525/files#r1726703685

File: packages/contentstack-bulk-publish/package.json [hunk 5]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@contentstack/cli-cm-bulk-publish",
   "description": "Contentstack CLI plugin for bulk publish actions",
-  "version": "1.4.8",
+  "version": "1.4.9",
```
Comment (aman19K) [inline packages/contentstack-bulk-publish/package.json:4]: @sunil-lakshman , https://github.com/contentstack/cli/pull/1525/files#r1726703685

File: packages/contentstack-branches/package.json [hunk 6]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@contentstack/cli-cm-branches",
   "description": "Contentstack CLI plugin to do branches operations",
-  "version": "1.1.2",
+  "version": "1.1.3",
```
Comment (aman19K) [inline packages/contentstack-branches/package.json:4]: @sunil-lakshman , https://github.com/contentstack/cli/pull/1525/files#r1726703685

File: packages/contentstack-bootstrap/package.json [hunk 7]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentstack/cli-cm-bootstrap",
   "description": "Bootstrap contentstack apps",
-  "version": "1.10.0",
+  "version": "1.10.1",
```
Comment (aman19K) [inline packages/contentstack-bootstrap/package.json:4]: @sunil-lakshman , https://github.com/contentstack/cli/pull/1525/files#r1726703685

File: packages/contentstack-auth/package.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@contentstack/cli-auth",
   "description": "Contentstack CLI plugin for authentication activities",
-  "version": "1.3.20",
+  "version": "1.3.21",
```
Comment (aman19K) [inline packages/contentstack-auth/package.json:4]: @sunil-lakshman , https://github.com/contentstack/cli/pull/1525/files#r1726703685

File: packages/contentstack-audit/package.json [hunk 9]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentstack/cli-audit",
-  "version": "1.6.5",
+  "version": "1.6.6",
```
Comment (aman19K) [inline packages/contentstack-audit/package.json:3]: @sunil-lakshman , https://github.com/contentstack/cli/pull/1525/files#r1726703685

**Review by cs-raj — APPROVED** [review]

**Review by aman19K — APPROVED** [review]

**Pull request merged**